lib/ASTGen/Sources/ASTGen/Exprs.swift

```diff
@@ -75,16 +72,16 @@ extension ASTGenVisitor {
     return .expr(sve)
   }
 
-  public func visit(_ node: TupleExprSyntax) -> ASTNode {
-    self.visit(node.elements, leftParen: node.leftParen, rightParen: node.rightParen)
+  public func generate(_ node: TupleExprSyntax) -> ASTNode {
+    self.generate(node.elements, leftParen: node.leftParen, rightParen: node.rightParen)
   }
 }
 
 extension ASTGenVisitor {
   /// Generate a tuple expression from a ``LabeledExprListSyntax`` and parentheses.
-  func visit(_ node: LabeledExprListSyntax, leftParen: TokenSyntax?, rightParen: TokenSyntax?) -> ASTNode {
+  func generate(_ node: LabeledExprListSyntax, leftParen: TokenSyntax?, rightParen: TokenSyntax?) -> ASTNode {
     let expressions = node.lazy.map {
-      self.visit($0.expression).rawValue
+        self.visit(Syntax($0.expression)).rawValue
```

Comment:
I think you applied 4 spaces of indentation here but the codebase is indented with 2 spaces.